svelte-kit-scss/src/lib/components/FileExplorer/FileExplorerAbout/FileExplorerAbout.svelte

```diff
@@ -1,10 +1,10 @@
 <script lang="ts">
   import { Book16, Link16 } from 'svelte-octicons';
   import { handleAnchorClick } from '$lib/helpers';
+  import type { RepoState } from '$lib/interfaces';
 
-  export let description: string;
-  export let website: string;
-  export let tags: string[];
+  export let repoInfo: RepoState;
+  const { description, website, tags } = repoInfo;
```

Comment:
In svelte components, destructuring like this will not be "reactive", so when, in this case if `description`, `website` or tags `tags` were to change, they would not update the elements and/components they are passed to in the template.

So in svelte components, you should (almost) always destructure using the Reactive Statements syntax, as [described in the svelte docs](https://svelte.dev/docs#component-format-script-3-$-marks-a-statement-as-reactive):
```js
$: ({ name } = person)
```

Otherwise it will surprise someone else someday, when things are not updating correctly.
This is very much only a svelte thing, but it is a very common svelte thing, and we should definitely follow the Svelte conventions for `starter.dev` projects